hsc/configs/hscconfig.yml

```diff
@@ -72,6 +72,18 @@ TXTruthLensSelector:
     r_i_cut: 100
     # may also need one for r_cpar_cut
 
+TXCosmosWeight:
+    mag_i_cut: 24.5
+    n_neighbors: 10
+
+TXPhotozDIR:
+    pz_code: 'ephor_ab'
+    pz_mark: 'best'
+    source_zbin_edges: [0.3, 0.6, 0.9, 1.2, 1.5]
+    lens_zbin_edges: [0.15,0.5,0.75,1.0,1.5]
```

Comment:
This means that I didn't update this config properly myself ¬¬. Could you please set the same source and lens bins?